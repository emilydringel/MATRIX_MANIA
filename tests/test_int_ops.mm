/* operators on ints */ 

def void testint(int a, int b){
    print(a + b);
    print(a - b);
    print(a * b);
    print(a / b);
    print(a == b);
    print(a == a);
    print(a != b);
    print(a != a);
    print(a > b);
    print(a >= b);
    print(a < b);
    print(a <= b);
}
def int main(){

    int c = 80;
    int d = 5;

    testint(c, d);

    testint(d, d);
}
