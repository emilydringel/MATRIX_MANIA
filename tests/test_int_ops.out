85
75
400
16
0
1
1
0
1
1
0
0